packages/cli/generators/relation/relationHasOne.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const RelationGenerator = require('./relation');
+
+module.exports = class RelationHasOne extends RelationGenerator {
+  constructor(args, opts) {
+    super(args, opts);
+  }
+
+  generateControllers(options) {
+    throw new Error('Not implemented');
+  }
+
+  generateModels(options) {
+    throw new Error('Not implemented');
+  }
+
+  generateRepositories(options) {
+    throw new Error('Not implemented');
```

Comment:
I assume these methods will be eventually implemented. Let's use a slightly different error message then:

```suggestion
    throw new Error('Not implemented yet');
```

It would be even better to point users to a GitHub issue keeping track of the task of implementing HasOne generator (replace 12345 with the real issue number):

```suggestion
    throw new Error('Not implemented yet, see https://github.com/strongloop/loopback-next/issues/12345');
```

Reply:
HasOne relation was removed from pool request. Not implemented yet.